lib/active_admin/application.rb

```diff
@@ -105,6 +105,9 @@ def initialize
     # Set flash message keys that shouldn't show in ActiveAdmin
     inheritable_setting :flash_keys_to_except, ['timedout']
 
+    # Set the order and column to be used for ordering for comments
+    inheritable_setting :comment_order, 'created_at ASC'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.